eea/reports/browser/report.pt

```diff
@@ -422,10 +397,20 @@
         <strong tal:content="here/Description" />
       </div>
 
+      <h2 tal:condition="python: not pdf_enabled" style="float:right;width:30%;" i18n:translate="">Download</h2>
```

Comment:
No inline styles, you have conditions, add a class that has these inline styles and instead toggle the class